jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/LambdaHandlerIndexTest.kt

```diff
@@ -74,7 +74,152 @@ class LambdaHandlerIndexTest {
+        }
+    }
+
+    @Test
+    fun testStreamHandlerIsReturned_customMethodInStreamRequestHandlerImplementation() {
+        val fixture = projectRule.fixture
+
+        fixture.openClass(
+            """
+            package com.example;
+
+            import com.amazonaws.services.lambda.runtime.Context;
+            import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+            import java.io.InputStream;
+            import java.io.OutputStream;
+            import java.io.IOException;
+
+            public class StreamLambdaHandler implements RequestStreamHandler {
+
+                @Override
+                public void handleRequest(InputStream input, OutputStream output, Context context) throws IOException {}
+
+                public String customHandleRequest(String request, Context context) {
+                    return request.toLowerCase();
+                }
+            }
+            """
+        )
+
+        runInEdtAndWait {
+            assertThat(LambdaHandlerIndex.listHandlers(projectRule.project))
+                    .contains("com.example.StreamLambdaHandler")
+                    .contains("com.example.StreamLambdaHandler::customHandleRequest")
+                    .doesNotContain("com.example.StreamLambdaHandler::handleRequest")
+        }
+    }
+
+    @Test
+    fun testHandlerIsReturned_fromSuperClass() {
+        val fixture = projectRule.fixture
+        fixture.openClass(
+            """
+            package com.example;
+
+            import com.amazonaws.services.lambda.runtime.Context;
+            import com.amazonaws.services.lambda.runtime.RequestHandler;
+
+            public abstract class AbstractClass implements RequestHandler<String, String> {
+
+                @Override
+                public String handleRequest(String request, Context context) {
+                    return internalHandleRequest(request, context);
+                }
+
+                public String customHandleRequest(String request, Context context) {
+                    return internalHandleRequest(request, context);
+                }
+
+                protected abstract String internalHandleRequest(String request, Context context);
+            }
+            """
+        )
+
+        fixture.openClass(
+            """
+            package com.example;
+
+            import com.amazonaws.services.lambda.runtime.Context;
+
+            public class ConcreteClass extends AbstractClass {
+
+                @Override
+                private String internalHandleRequest(String request, Context context) {
+                    return request.toUpperCase();
+                }
+            }
+            """
+        )
+
+        runInEdtAndWait {
+            assertThat(LambdaHandlerIndex.listHandlers(projectRule.project))
+                    .contains("com.example.ConcreteClass")
+                    .contains("com.example.ConcreteClass::customHandleRequest")
```

Comment:
Agree this was missing - that's the real problem right?

Reply:
Yeah, this CR is to solve this missing part.